collector/protocols/modbus.md

```diff
@@ -19,11 +20,40 @@ There are a number of memory areas defined in the MODBUS specification. Here, ar
   * Input Register - Short input value, usually representing an analog input to the PLC
   * Holding Register - Short value, usually representing an analog output from the PLC
 
+## Configuring the `endpoint` field for Modbus sources
+### LiveRig Collector 5.0.0 or above
+If you are using a LiveRig version 5.0.0 or above, you can change the connection protocol by declaring the endpoint using the
+format: `{code}:{transport}://{ip-address}:{port}?{options}`. Here is a list of options:
+
+- Code: `modbus-tcp`*(default)*, `modbus-adu`, `modbus-ascii`;
+- Transport: `tcp`*(default)*, `udp`, `serial`;
+- Options: `request-timeout`, `unit-identifier` (This will use the following
+  structure: `?{option-1-name}={option-1-value}&{option-2-name}={option-2-value}`)
```

Comment:
```suggestion
- **code**: `modbus-tcp`*(default)*, `modbus-adu`, `modbus-ascii`;
- **transport**: `tcp`*(default)*, `udp`, `serial`;
- **options**: `request-timeout`, `unit-identifier` (This will use the following structure: `?{option-1-name}={option-1-value}&{option-2-name}={option-2-value}`)
```